Source/Core/VideoCommon/OpcodeDecoding.cpp

```diff
@@ -213,11 +207,19 @@ class RunCallback final : public Callback
     }
     else
     {
-      if (!s_is_fifo_error_seen)
+      // Datel software uses 0x01 during startup, and Mario Party 5's Wiggler capsule
+      // accidentally uses 0x01-0x03 due to sending 4 more vertices than intended.
+      // Hardware testing indicates that 0x01-0x07 do nothing, so to avoid annoying the user with
+      // spurious popups, we don't create a panic alert in those cases.  Other unknown opcodes
+      // (such as 0x18) seem to result in hangs.
+      if (!s_is_fifo_error_seen && opcode > 0x07)
+      {
         CommandProcessor::HandleUnknownOpcode(opcode, data, is_preprocess);
+        s_is_fifo_error_seen = true;
+      }
+
       ERROR_LOG_FMT(VIDEO, "FIFO: Unknown Opcode({:#04x} @ {}, preprocessing = {})", opcode,
```

Comment:
It does seem a little deceptive that this log is printed outside the 'handleunknownopcode' trigger.

Reply:
I'm assuming this was intentional

Reply:
It's intentional in that we always want the log to be printed, but I don't think there really is any reason we can't move the logging and `s_is_fifo_error_seen` into `CommandProcessor`.